src/compiler/reviews.md

```diff
@@ -1,55 +1,388 @@
+     fixes.
+  2. For the rationale to be even more useful, authors are encouraged to list
+     potential **points of contention**, **compromises** that needed to be made,
+     **alternative approaches** that have been considered, **relevant
+     documentation, discussions and context**, etc.
+    - Reviewing code is difficult, and reviewers only have a **limited amount of
+      time** to do it. Jump-starting the review process by not making the
+      reviewer puzzle together the intention and context of a pull request will
+      not only speed things up but also improve the quality of the review.
+- Reviewers must have a good idea on whether they are the **right person to
+  approve** the change.
+  - Knowledge of the code under review is an obvious but not the only criteria
+    for answering this question.
+  - Procedure wise, the reviewer also needs to decide:
+    - Can the reviewer make the decision alone?
+    - Does the PR needs to go through the [Major Change Process][mcp], or does
+      it need a T-compiler [Final Comment Period][fcp] sign-off, or does it need
+      a full [Request For Comments (RFC)][rfc]?
+    - Does the PR need reviews and/or sign-off from other teams, particularly
+      T-lang?
+    - Can the changes break stable code or begin accepting new code that we do
+      not intend to? If the PR contains risks, is it sufficiently justified?
+      Does the changes need ecosystem impact evaluation through crater runs?
+    - Will the PR introduce significant perf changes? If there might be a perf
+      regression, is it justified? Does the PR need a perf run? 
+    - Can the reviewer perform the review sufficiently thorough and in a timely
+      fashion?
+    - Is the reviewer impartial enough to provide a sufficiently unbiased
+      perspective? E.g. due to co-authorship (sufficiently significant changes
+      to the PR made by the reviewer) or other conflicts-of-interest?
+
+[rfc]: https://github.com/rust-lang/rfcs
+
+## Reviewing Checklist
+
+The following list of questions will help reviewers and PR authors alike to
+bring PRs into good shape and meet the above criteria:
+
+> #### Checklist for PR authors and reviewers
+> * Does the PR message have
+>    * A concise **high-level description** of the changes? (*what* is being
+>      changed)
+>    * A clear **rationale** for doing them? (*why* is it being changed)
+>    * If non-trivial and if suitable, **how** the bug is fixed or **how** the
+>      change is implemented?
+>    * A list of potential **points of contention**? Alternatives? Trade-offs?
+>      Risks?
+>    * **Links to relevant issues**, RFCs, MCPs, etc?
+> * Does the PR need a **regression test**? Does the PR have **sufficient test
+>   coverage**?
+> * Does the change need to be covered by a **[major change proposal][mcp]**? Is
+>   it already covered? If there is already a MCP open, was it already accepted,
+>   or is the PR blocked on that?
+> * Does the PR need a **pe
```

Comment:
Is something missed here?

Reply:
Yes, good catch, this was supposed to be 'perf run'

Reply:
Fixed